base_user_empty_password/wizard/empty_password.py

```diff
@@ -1,22 +1,25 @@
 # Copyright 2024 Camptocamp SA
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
-from odoo import fields, models
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
 
 
 class EmptyPasswordWizard(models.TransientModel):
     _name = "empty.password.wizard"
     _description = "Empty Password Wizard"
 
-    user_ids = fields.Many2many(
-        "res.users", readonly=True, default=lambda w: w._default_user_ids()
-    )
+    user_ids = fields.Many2many("res.users", readonly=True)
```

Comment:
> If a field is marked as `readonly=True` on the model, it won't be possible to import it through the Odoo import tool, so avoid it as possible and define the **readonly attribute on the views instead.**

Reply:
thanks @xaviedoanhduy , I have updated it

Reply:
`<field name="user_ids" invisible="1" />` so we don't need to add readonly attribute

Reply:
Thank you for your attention, GLTM overall

Reply:
>  it won't be possible to import it through the Odoo import tool
why would we care about importing records for this wizard?

Reply:
I understand, but this guideline doesn't really make sense for wizards; I don't think we should change that part, this will just introduce confusion

Reply:
@chaule97 can you restore that part please?

Reply:
you just restored the readonly but what's the added value of the `default_get`?

Reply:
selected user 
![image](https://github.com/user-attachments/assets/3eecca14-4558-4b35-a6f3-b824e48b495e)
before, this added value in [_default_user_ids](https://github.com/OCA/server-auth/blob/c6a446296ae9ade455276255412996064adaa77d/base_user_empty_password/wizard/empty_password.py#L14)

Reply:
but the result is exactly the same, no?

Reply:
yes, same. I just updated it according to odoo 18 sample code